events/models.py

```diff
@@ -49,10 +49,10 @@ def get_context(self, request, *args, **kwargs):
         archive_years = EventPage.objects.live().descendant_of(self).filter(date_end__lte=now).dates('date_end', 'year', order='DESC')
         past = request.GET.get('past') == "1"
         if past:
-            filter_dict["date_end__lte"] = now
+            filter_dict["date_end__lt"] = now  # an event is in the past if it ended "yesterday"
```

Comment:
> # an event is in the past if it ended "yesterday"

^^ This isn’t a date comparison, it’s a _datetime_ comparison, so while the comment is true, "yesterday" here is a bit misleading. This filter gets all events that ended at least one second ago. So this end date could be today, but in the past.